### PR TITLE
Removed Profile V1

### PR DESCRIPTION
**Pull request by Mahadev-Vasireddy**

### CODE REVIEW

File: src/britive/britive.py
Comment (Mahadev-Vasireddy) [inline src/britive/britive.py:1]: Same as your change

File: src/britive/profiles.py [hunk 1]
```diff
@@ -22,27 +22,20 @@
 
 
 class Profiles:
-    def __init__(self, britive, version: int = 2):  # default to profiles v2 as v2 will be enabled for new tenants
+    def __init__(self, britive):
         self.britive = britive
         self.base_url = f'{self.britive.base_url}/apps'
-        self.version = version
+        # self.version = version
```
Comment (Mahadev-Vasireddy) [inline src/britive/profiles.py:28]: version is not a class variable anymore. Need to remove, but commented for now

File: src/britive/profiles.py [hunk 2]
```diff
@@ -22,27 +22,20 @@
...
 
-        if self.version == 1:
-            self.identities = ProfileIdentities(britive)
-            self.tags = ProfileTags(britive)
-        elif self.version == 2:
-            self.policies = ProfilePolicies(britive)
-        else:
-            raise Exception(f'invalid profile version - {self.version} - cannot continue')
+        self.policies = ProfilePolicies(britive)
+
```
Comment (Mahadev-Vasireddy) [inline src/britive/profiles.py:34]: not sure of this change

File: tests/cache.py [hunk 3]
```diff
@@ -11,7 +11,7 @@
 
 
 britive = Britive()  # source details from environment variables
-profiles_v1 = britive.feature_flags['profile-v1']
+profiles_v1 = False
```
Comment (Mahadev-Vasireddy) [inline tests/cache.py:14]: I have to check, but should we remove these 4 variables?

File: tests/cache.py [hunk 4]
```diff
@@ -378,25 +378,15 @@ def cached_checked_out_profile(pytestconfig, cached_profile, cached_user, cached
...
-            description=cached_tag['name'],
-        )
-        britive.profiles.policies.create(
-            profile_id=cached_profile['papId'],
-            policy=policy
-        )
+    policy = britive.profiles.policies.build(
+        name=cached_profile['papId'],
+        users=[calling_user_details['username']],
+        description=cached_tag['name'],
```
Comment (Mahadev-Vasireddy) [inline tests/cache.py:384]: Kept only the V2 logic

File: tests/test_130-profiles.py [hunk 5]
```diff
@@ -177,52 +147,16 @@ def test_session_attributes_remove(cached_profile, cached_static_session_attribu
...
-
-
-@pytest.mark.skipif(profiles_v1, reason=profile_v1_skip)
 def test_policies_create(cached_profile_policy):
     assert isinstance(cached_profile_policy, dict)
     assert cached_profile_policy['members']['tags']
 
 
-@pytest.mark.skipif(profiles_v1, reason=profile_v1_skip)
 def test_policies_create_with_approval_single_notification_medium(cached_profile_approval_policy):
```
Comment (Mahadev-Vasireddy) [inline tests/test_130-profiles.py:155]: Tests are same as your branch

File: tests/test_130-profiles.py
Comment (Mahadev-Vasireddy) [inline tests/test_130-profiles.py:1]: Changes to tests are same

File: tests/cache.py (hunk 3 above)
Comment (twratl) [reply tests/cache.py:14]: yes, lets remove all references to profile v1 and v2 in the test code.

**Pull request closed without merging**